src/utils/isReactComponent.js

```diff
@@ -1,14 +1,12 @@
 export default function isReactComponent (component) {
-  if (typeof component === 'object') {
+  if (typeof component === 'object' && component.styledComponentId === undefined) {
```

Comment:
This hack doesn't look good... Are you sure there's no better way to detect a styled component?

Also, if I'm not mistaken, React cannot render regular objects at all (if that hasn't been changed recently), so there should be another way.